lib/id3c/cli/redcap.py

```diff
@@ -289,7 +289,7 @@ def _fetch_records(self, parameters: dict) -> Iterator['Record']:
         return (Record(self, r) for r in self._fetch("record", parameters))
 
 
-    def update_records(self, records: List[Dict[str, str]]) -> int:
+    def update_records(self, records: List[Dict[str, str]], date_format: str = "YMD") -> int:
```

Comment:
The EHS pipeline does create new records in a redcap project. Will update_records() work for that use case? The comments call out that this is for updating existing records and "not suitable for creating new records in projects that use auto-numbered record ids.", but that project uses barcodes as record ids so that exclusion may not apply.

Reply:
I believe it is suitable because auto-numbered ids aren't in use, but have not tested it directly yet because I'm not sure how to do that without doing it against the production project.

I documented the auto-numbered ids caveat because at the time I added `update_records()` I didn't want to think about the edge cases there (with regard to `forceAutoNumbering: false` and other things). I started too, but then decided that was scope creep I didn't want to deal with at the time.